test/controllers/update/overseas.entity.update.details.controller.spec.ts

```diff
@@ -3,6 +3,7 @@ jest.mock("../../../src/utils/logger");
 jest.mock('../../../src/middleware/authentication.middleware');
```

Comment:
I would have like a POST test to see it re-directs if there is no entity_number to check that the navigation control is in place.